webapi/src/it/scala/org/knora/sipi/SipiIT.scala

```diff
@@ -29,70 +32,113 @@ import org.knora.webapi.testcontainers.SipiTestContainer
 
 object SipiIT extends ZIOSpecDefault {
 
-  private val identifierTestFile  = "250x250.jp2"
-  private val prefix              = "0001"
-  private def copyTestImageToSipi = SipiTestContainer.copyImageToContainer(prefix, identifierTestFile)
-  private def sendGetRequestToSipi(path: String) =
+  private val imageTestfile = "FGiLaT4zzuV-CqwbEDFAFeS.jp2"
+  private val infoTestfile  = "FGiLaT4zzuV-CqwbEDFAFeS.info"
+  private val origTestfile  = "FGiLaT4zzuV-CqwbEDFAFeS.jp2.orig"
+  private val prefix        = "0001"
+  private def copyTestFilesToSipi = ZIO.foreach(List(imageTestfile, infoTestfile, origTestfile))(
+    SipiTestContainer.copyFileToImageFolderInContainer(prefix, _)
+  )
+
+  private def getWithoutAuthorization(path: String) =
     SipiTestContainer.resolveUrl(path).map(Request.get).flatMap(Client.request(_))
 
+  private val knoraJsonEndpointSuite =
+    suite("Endpoint /{prefix}/{identifier}/knora.json")(
+      suite("Given the user is unauthorized")(
+        suite("And given a .info file exists in Sipi")(
```

Comment:
Just wonder if this test description could/should be in one suite, since there is only on test here?